client/src/components/screens/PatientSubmission/index.jsx

```diff
@@ -449,6 +449,11 @@ function PatientSubmissionScreen(props) {
       }
 
       actions.createRequest(batch);
+      if (submitted) {
+        setTimeout(() => {
+          actions.navigateToPatientSearchScreen();
+        }, 600);
```

Comment:
Ça sens le problème futur mais jte trust pour la démo lol

Reply:
Je comptais faire ça après la démo, mais here you go lol